packages/pcd/semaphore-identity-pcd/src/forwardedTypes.ts

```diff
@@ -0,0 +1 @@
+export { Identity as IdentityV3 } from "@semaphore-protocol/identity";
```

Comment:
Interesting approach to the 2-versions-of-the-same-package problem.
Richard pointed out another one in package.json, I believe.
Also Cedoor pointed out there's a @semaphore/core package which re-exports other ones, which could be used to avoid confusion.
No suggested change here, just pointing out multiple alternatives as I learn.